ingestion_server/test/unit_test.py

```diff
@@ -140,9 +140,9 @@ def test_url_protocol_fix():
         result = CleanupFunctions.cleanup_url(bad_url, tls_support_cache)
         expected = "'https://flickr.com'"
 
-        bad_http = "neverssl.com"
+        bad_http = "baidu.com"
         result_http = CleanupFunctions.cleanup_url(bad_http, tls_support_cache)
-        expected_http = "'http://neverssl.com'"
+        expected_http = "'http://baidu.com'"
```

Comment:
I can speculate as to why, but what's the reason for the domain switch here?

Reply:
NeverSSL now supports SSL ☹️

Reply:
Another one I know is http://mitm.cool (`man in the middle` lol)

Reply:
That's cool but mocking is the stablest solution.